samples/samples.mjs

```diff
@@ -23,4 +23,7 @@ export default [
     { title: "Hidden Shift", file: "./algorithms/HiddenShiftNISQ.qs", shots: 1 },
     { title: "Hidden Shift (Advanced)", file: "./algorithms/HiddenShift.qs", shots: 1 },
     { title: "Shor", file: "./algorithms/Shor.qs", shots: 1 },
+    { title: "Dynamics", file: "./estimation/Dynamics.qs", shots: 1 },
+    { title: "Precalculated", file: "./estimation/Precalculated.qs", shots: 1 },
+    { title: "Shor's Resource Estimation", file: "./estimation/ShorRE.qs", shots: 1 },
```

Comment:
since these won't show up in a different subfolder, I think giving them a name that differentiates them would be helpful (like the "(Advanced)" used above):

```suggestion
    { title: "Dynamics (Resource Estimation)", file: "./estimation/Dynamics.qs", shots: 1 },
    { title: "Precalculated (Resource Estimation)", file: "./estimation/Precalculated.qs", shots: 1 },
    { title: "Shor (Resource Estimation)", file: "./estimation/ShorRE.qs", shots: 1 },
```